getSigmaPointApp.m

```diff
@@ -178,7 +195,7 @@
     case 'dmd'
         diracD = size(D,1);
         [SPToolboxFolder,~,~] = fileparts(which('CompDMD_Location'));
-        filename = sprintf('%s%i%s%i%s','DMDinfo\B_SP_dim',diracD,'points',op_SP.n_samples,'.csv');
+        filename = sprintf('%s%i%s%i%s','DMDTrueMeaninfo\B_SP_dim',diracD,'points',op_SP.n_samples,'.csv');
```

Comment:
Here again, watch out with the filename:
This path information using "\" is only valid on Windows. You could use an "if/else" or work with the "fileparts" command to retrieve other files from the Matlab path. They should anyhow be part of the path...